tools/polar_import.py

```
@@ -116,7 +116,7 @@ def get_current_xc_polar(glider, weight=0.0):
         return None
     ret = None
     for line in f:
-        if line[0:7] != "  { " : continue
+        if line[0:4] != "  { " : continue
```

Ruff (0.15.1)

[error] Multiple statements on one line (colon)

(E701)